cub/test/catch2_test_block_scan.cu

```diff
@@ -185,7 +233,7 @@ struct sum_aggregate_op_t
   {
     T block_aggregate{};
 
-    if (Mode == scan_mode::exclusive)
+    if constexpr (Mode == scan_mode::exclusive)
```

Comment:
Thanks for adding these 🫶 Came across these the other day and had a mental note to make them `constexpr`. Now it's done already. Thanks!